### PR TITLE
[new] kurento-client v6.0.0

### PR DESCRIPTION
**Pull request by paleozogt**
JavaScript API for the [Kurento WebRTC media server](http://www.kurento.org/).

The source-code is [here](https://github.com/Kurento/kurento-client-js/tree/6.0.0). The minified js file was obtained via npm:

```
npm install kurento-client
```

### CODE REVIEW

File: ajax/libs/kurento-client/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "basePath": "/src/main/resources/META-INF/resources/js/",
+      "files": [
+        "kurento*"
+      ]
+    }
+  ],
+  "filename": "kurento-client.min.js",
+  "version": "6.0.0",
+  "description": "Media API for the Kurento Web SDK",
+  "homepage": "http://www.kurento.org",
```
Comment (pvnr0082t) [inline ajax/libs/kurento-client/package.json]: It's better to use `https` instead of `http`.

File: ajax/libs/kurento-client/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "keywords": [
+    "API",
+    "kurento",
+    "WebRTC"
+  ],
+  "maintainers": [
```
Comment (pvnr0082t) [inline ajax/libs/kurento-client/package.json]: Let's remove `"maintainers"`, `"bugs"` field.

File: ajax/libs/kurento-client/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      "name": "kurento-maintainer-team",
+      "email": "info@kurento.org"
+    }
+  ],
+  "author": "Kurento <info@kurento.com>",
+  "bugs": {
+    "email": "kurento@googlegroups.com"
+  },
+  "repositories": [
```
Comment (pvnr0082t) [inline ajax/libs/kurento-client/package.json]: Let's replace `"repositories"` with `"repository"` as follow:
```json
  "repository": {
    "type": "git",
    "url": "https://github.com/Kurento/kurento-client-js"
  },
```

File: ajax/libs/kurento-client/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      "type": "git",
+      "url": "https://github.com/Kurento/kurento-client-js"
+    }
+  ],
+  "license": "LGPL-2.1"
```
Comment (pvnr0082t) [inline ajax/libs/kurento-client/package.json]: The latest version changed license to `Apache-2.0`. Ref: https://github.com/Kurento/kurento-client-js/blob/master/LICENSE

File: ajax/libs/kurento-client/package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "kurento-client",
+  "npmName": "kurento-client",
+  "npmFileMap": [
+    {
+      "basePath": "/src/main/resources/META-INF/resources/js/",
```
Comment (pvnr0082t) [inline ajax/libs/kurento-client/package.json]: Let's remove the leading and trailing slash of basePath like `"src/main/resources/META-INF/resources/js"`.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**